dev/src/components/ReviewPage.tsx

```diff
@@ -65,10 +68,11 @@ const ReviewPage = (): JSX.Element => {
                 <div className={state.review.assignmentID >= 0 ? "col-md-4" : "col-md-6"}>
                     <Search placeholder={"Search by name ..."} >
                         <Button type={ButtonType.BUTTON}
-                            text={groupView ? "View by group" : "View by student"}
-                            onclick={() => { setGroupView(!groupView); actions.review.setAssignmentID(-1) }}
+                            text={groupView ? "View by student" : "View by group"}
```

Comment:
Maybe the string `View by ` can be moved first and concatenate with `groupView ? "student" : "group"`

BTW: Is it correct that `groupView === true` should display `student`?

Reply:
I will concatenate the string.

Regarding `groupView === true -> student`: I'm not entirely sure what makes the most sense.

The change makes the button tell the user what will happen when s/he clicks the button. The previous behaviour told the user what s/he was currently viewing.

i.e. the button now communicates the resulting action of clicking it, rather than the current state.

Reply:
Good point, didn’t think about that